### DISCUSSION TITLE
How to use "patched" version ?

### DISCUSSION THREAD

**Discussion by PaoRyuu** [humhue/DAOTools#11] [Q&A] [answered]
Hi,

First of all let me thank you for going through the effort and releasing this fix to fix :)

I am completely not trained in coding etc. so I tried using your "patched" version of files that are submitted in release section but I don't think that they did the trick, unless I do not understand the purpose of the fix. 

Can you explain, if your patch is supposed to provide all QUDAO dialogue fixed in localized language, based on what you choose in your DA:O starter? For example, I play in Polish and all is well and fine with Polish language and dialogues but when I install QUDAO some dialogues are changing to English (for example Alistair) which is very annoying and I was hoping your file is fixing.

What I did, I did download your QUDAO v 3.5 patched, I uninstalled my original QUDAO and installed your patched version via DA:O Mod Manager, but I still have most dialogues in English vs Polish.

Thank you for your answer,

Pao

**Comment by humhue** [comment, marked as answer]
Does the Polish version have a voice over as well? If that's the case, you'd have to remove all .fsb and .fev files from the the sound directory in [...]\qwinn_fixpack_3\core\audio\sound inside of the .dazip file and then zip that again. If the Polish version uses the English voice over, you shouldn't notice anything weird. To put it in another way: this patch reverts most of the subtitles and in-game texts to their original non-english version, but as of now doesn't touch the audio files Qwinn himself added.

**Comment by PaoRyuu** [comment]
Thanks!

Yes indeed, there is Polish voiceover which is broken.

I actually removed fsb and fev files from add-ins already after
installation of QUDAO and it works fine now!

Thanks a lot for your help!

**Comment by humhue** [comment]
I'm glad that it's working for you :)
